draft-ietf-regext-rdap-geofeed.md

```diff
@@ -48,6 +48,11 @@ The keywords "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SH
 "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in BCP 14
 [@!RFC2119] [@!RFC8174] when, and only when, they appear in all capitals, as shown here.
 
+Indentation and whitespace in examples are provided only to illustrate element relationships, and are not a REQUIRED
+feature of this protocol.
+
+"..." in examples is used as shorthand for elements defined outside of this document.
```

Comment:
Sorry, shamelessly stole your verbiage from the RIR Search draft. :)

Reply:
Which in turn came from 8334. :)